governance/votes/EXERCISE - Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,214 @@
+| Navigator        | [0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) | 0.32         |
+
+
+
+### CRVV1ETHSTETH-A - 1st Stage Offboarding
+
+Per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#14-3-12), the responsible Facilitator has proposed ([here](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238#crvv1ethsteth-a-proposed-offboarding-11) and [here](http://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6?u=le_bateleur)) taking the first step to offboard CRVV1ETHSTETH-A. The Stability Scope provides that certain collateral types must be offboarded if they fall below a total debt of 20 million. The total debt from CRVV1ETHSTETH-A fell below this threshold on June 19, 2023 and has not recovered. Until CRVV1ETHSTETH-A can be fully offboarded, the Facilitator proposes that the initial step be taken of setting the CRVV1ETHSTETH-A Debt Ceiling to **zero Dai**, thus precluding further minting. If this executive proposal passes, the following actions will occur:
+
+- CRVV1ETHSTETH-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) `(line)` will be set to **zero Dai**.
+- CRVV1ETHSTETH-A will be removed from [Debt Ceiling Instant Access](https://manual.makerdao.com/module-index/module-dciam) (`MCD_IAM_AUTO_LINE`).
+
+### Ecosystem Actor Funding
+
+#### A) Ecosystem Actor Dai Budget Streams
+
+Per the [Support Scope](https://mips.makerdao.com/mips/details/MIP106#7-4-2) and [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-launch-project), the proposed actions concern setting up Dai budget streams to fund the work of Ecosystem Actors Jetstream and Chronicle Labs. 
+
+- [Jetstream](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-jetstream/21054) takes over the existing duties of the Development & UX (DUX-001) Core Unit and will be initially involved in the [Launch Project](http://forum.makerdao.com/t/the-launch-project-and-scope-artifact-update-proposal/21048?u=le_bateleur). The Accessibility Scope Facilitator has [approved](http://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306/2?u=le_bateleur) the onboarding of Jetstream as an Ecosystem Actor.
+
+- Spinning off from the Oracles Core Unit, [Chronicle Labs](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-chronicle-labs/21053) will launch a decentralized oracle protocol and infrastructure services to support the soon-to-launch SubDAOs. In a recent [poll](https://vote.makerdao.com/polling/QmdnSKPu), Maker governance ratified amendments to the Support Scope that include a [provision incubating Chronicle Labs](https://mips.makerdao.com/mips/details/MIP106#7-4-2-1a) with a one-year linear stream.  
+
+If this executive proposal passes, the following Dai budget streams will be executed:
```

Comment:
Executed is the wrong term to use with budget streams. It implies a single one-and-done action, which is the opposite of what the streams are. Execute transfers, set up or authorize streams.